src/open_inwoner/cms/products/cms_plugins.py

```diff
@@ -34,8 +34,17 @@ def render(self, context, instance, placeholder):
             .filter(highlighted=True)
             .order_by("path")
         )
-        if request.user.is_authenticated and request.user.selected_categories.exists():
-            categories = request.user.selected_categories.order_by("name")[: self.limit]
+        if request.user.is_authenticated and request.user.bsn:
+            categories = (
+                Category.objects.published()
+                .filter_for_user_with_zaken(request.user)
+                .order_by("name")[: self.limit]
+            )
+
+            # Fallback for if the user has no cases with case types that are linked to
+            # categories
+            if not categories:
```

Comment:
@alextreme I added this because I assumed it wouldn't be right to show no categories at all in this case

Reply:
Falling back to the current behaviour is fine